sjb/config/test_cases/test_branch_origin_aggregated_logging_prior.yml

```diff
@@ -1,6 +1,7 @@
 ---
 parent: 'common/test_cases/origin_built_release.yml'
```

Comment:
@dobbymoodge you might be able to make this diff a lot smaller by changing the parent templates

Reply:
@stevekuznetsov I tried to trace out which templates I could tweak and only affect these particular tests, and there's enough overlap with other tests that I didn't want to mess with it.

Reply:
OK, sounds good